Doc/c-api/unicode.rst

```diff
@@ -536,6 +540,9 @@ APIs:
    .. [1] For integer specifiers (d, u, ld, li, lu, lld, lli, llu, zd, zi,
       zu, i, x): the 0-conversion flag has effect even when a precision is given.
 
+   .. [2] The object type fully qualified name is equivalent to:
+      ``f"{type(obj).__module__}.{type(obj).__qualname__}"``.
```

Comment:
Except that the module name is omitted for types in the builtins module (and for non-heap extension types that don't specify the module, but this can be considered as a bug).